app/src/main/java/org/wikipedia/analytics/eventplatform/MachineGeneratedArticleDescriptionsAnalyticsHelper.kt

```diff
@@ -21,4 +23,64 @@ object MachineGeneratedArticleDescriptionsAnalyticsHelper {
             )
         )
     }
+
+    fun suggestedDescriptionsButtonShown(context: Context) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "$MACHINE_GEN_DESC_SUGGESTIONS.suggestedDescriptionsButton.shown"
+            )
+        )
+    }
+
+    fun machineGeneratedSuggestionsDetailsLogged(context: Context, articleName: String,
+                                                 suggestionsList: List<String>, isBlp: Boolean) {
+        EventPlatformClient.submit(
+            BreadCrumbLogEvent(
+                BreadCrumbViewUtil.getReadableScreenName(context),
+                "$MACHINE_GEN_DESC_SUGGESTIONS.ApiResponseDetails.articleName:$articleName.isBlp:$isBlp.NumberOfSuggestionsOffered:${suggestionsList.size}.Suggestions:${
+                    suggestionsList.joinToString(",")
```

Comment:
Feels like we can combine the list items to a string before submitting the event.

Would it be possible to set a `val suggestions = suggestionsList.joinToString(",")` before `submit()`?